package.json

```diff
@@ -56,7 +55,7 @@
         "@babel/plugin-proposal-export-default-from": "7.8.3",
         "@babel/plugin-transform-modules-commonjs": "7.8.3",
         "@babel/plugin-transform-runtime": "7.8.3",
-        "@babel/preset-env": "7.8.4",
+        "@babel/preset-env": "^7.8.7",
```

Comment:
To test whether all of my stuff worked, I removed everything and reinstalled. Then got a babel error. I googled it and they fixed a bug two weeks ago. Hence the update.